libkbfs/md_ops.go

```diff
@@ -221,6 +221,13 @@ func (md *MDOpsStandard) verifyKey(
 				"Skipping revoked key verification due to recursion")
 			return false, nil
 		}
+		if e.info.MerkleRoot.Seqno <= 0 {
+			md.log.CDebugf(ctx, "Can't verify an MD written by a revoked "+
+				"device if there's no valid root seqno to check: %d",
+				e.info.MerkleRoot.Seqno)
+			return false, err
```

Comment:
It seems returning `err` here cause downstream failures because we can't verify keys. Do we not have legitimate revokes from earlier that didn't include seqno?

Reply:
@songgao: I don't follow what "downstream faillures" you're talking about.  But yes, as I said in the description, the API server returns 0 for some revokes, and that's a known issue that we have to work around.  I'll be posting another PR soon to decrease reliance on the field that can be 0.

Reply:
Sorry downstream is probably not the right term. What I meant was when we return an error in this case, whatever that called `verifyKey` would get an error. Doesn't that cause read failures?

Reply:
Yes, but we were already returning an error in that case, because `FindNextMD` fails with an error instead (see keybase/client#12101).  This just gives a better error message and fixes the QR problem.

Reply:
Ah I see; I misunderstood this PR as the fix making it work. Thanks.